README.md

```diff
@@ -421,7 +420,7 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 - Open your app that you created in [Create a Function App](#create-a-function-app) in VSCode
 - Press 'F1' and search for 'Azure Functions: Create Function'
 - Choose HttpTrigger -> (Provide a function name) -> anonymous
-- In the file that opens (__init__.py), replace the 'def main(req: func.HttpRequest) -> func.HttpResponse:' block with the below code.
+- In the file that opens (\_\_init\_\_.py), replace the 'def main(req: func.HttpRequest) -> func.HttpResponse:' block with the below code.
```

Comment:
Same here with the backticks - I usually try to use that for paths as well.

(and then the python code fragment as well)